src/main/java/org/candlepin/auth/CloudAuthTokenType.java

```diff
@@ -0,0 +1,18 @@
+package org.candlepin.auth;
+
+// TODO: Java Docs, does this enum need to be generated by open api spec?
```

Comment:
Nope. We probably also don't need the enum at all if we're separating out the two auth handlers. We likely won't need to be making the comparison enough to justify parsing it into a full object.